core/app/c/[communitySlug]/settings/tokens/ExistingToken.tsx

```diff
@@ -15,7 +16,16 @@ export const ExistingToken = ({ token }: { token: SafeApiAccessToken }) => {
 					)}
 				</div>
 				<div className="flex items-center justify-end">
-					<RevokeTokenButton token={token} />
+					{token.isSiteBuilderToken ? (
+						<InfoButton>
+							<p>
+								This token is a site builder token. It has read-only access to all
+								content in the community. It cannot be revoked.
+							</p>
+						</InfoButton>
+					) : (
+						<RevokeTokenButton token={token} />
+					)}
```

Comment:
we should be able to regen it though, in case it gets leaked

- [ ] add ability to regenerate site builder token